manifests/ca.pp

```diff
@@ -104,6 +107,7 @@
   $key_name     = '',
   $key_ou       = '',
   $tls_auth     = false,
+  $easyrsa_source = $::openvpn::params::easyrsa_source,
```

Comment:
We're still on 12.04. Now i see what you're requesting, i'll take a second look at it!